trading_calendars/exchange_calendar_xkrx.py

```diff
@@ -0,0 +1,546 @@
+    "2010-09-22",
+    "2010-09-23",
+    "2010-12-31",
+    "2011-02-02",
+    "2011-02-03",
+    "2011-02-04",
+    "2011-03-01",
+    "2011-05-05",
+    "2011-05-10",
+    "2011-06-06",
+    "2011-08-15",
+    "2011-09-12",
+    "2011-09-13",
+    "2011-10-03",
+    "2011-12-30",
+    "2012-01-23",
+    "2012-01-24",
+    "2012-03-01",
+    "2012-04-11",
+    "2012-05-01",
+    "2012-05-28",
+    "2012-06-06",
+    "2012-08-15",
+    "2012-10-01",
+    "2012-10-03",
+    "2012-12-19",
+    "2012-12-25",
+    "2012-12-31",
+    "2013-01-01",
+    "2013-02-11",
+    "2013-03-01",
+    "2013-05-01",
+    "2013-05-17",
+    "2013-06-06",
+    "2013-08-15",
+    "2013-09-18",
+    "2013-09-19",
+    "2013-09-20",
+    "2013-10-03",
+    "2013-10-09",
+    "2013-12-25",
+    "2013-12-31",
+    "2014-01-01",
+    "2014-01-30",
+    "2014-01-31",
+    "2014-05-01",
+    "2014-05-05",
+    "2014-05-06",
+    "2014-06-04",
+    "2014-06-06",
+    "2014-08-15",
+    "2014-09-08",
+    "2014-09-09",
+    "2014-09-10",
+    "2014-10-03",
+    "2014-10-09",
+    "2014-12-25",
+    "2014-12-31",
+    "2015-01-01",
+    "2015-02-18",
+    "2015-02-19",
+    "2015-02-20",
+    "2015-05-01",
+    "2015-05-05",
+    "2015-05-25",
+    "2015-08-14",
+    "2015-09-28",
+    "2015-09-29",
+    "2015-10-09",
+    "2015-12-25",
+    "2015-12-31",
+    "2016-01-01",
+    "2016-02-08",
+    "2016-02-09",
+    "2016-02-10",
+    "2016-03-01",
+    "2016-04-13",
+    "2016-05-05",
+    "2016-05-06",
+    "2016-06-06",
+    "2016-08-15",
+    "2016-09-14",
+    "2016-09-15",
+    "2016-09-16",
+    "2016-10-03",
+    "2016-12-30",
+    "2017-01-27",
+    "2017-01-30",
+    "2017-03-01",
+    "2017-05-01",
+    "2017-05-03",
+    "2017-05-05",
+    "2017-05-09",
+    "2017-06-06",
+    "2017-08-15",
+    "2017-10-02",
+    "2017-10-03",
+    "2017-10-04",
+    "2017-10-05",
+    "2017-10-06",
+    "2017-10-09",
+    "2017-12-25",
+    "2017-12-29",
+    "2018-01-01",
+    "2018-02-15",
+    "2018-02-16",
+    "2018-03-01",
+    "2018-05-01",
+    "2018-05-07",
+    "2018-05-22",
+    "2018-06-06",
+    "2018-06-13",
+    "2018-08-15",
+    "2018-09-24",
+    "2018-09-25",
+    "2018-09-26",
+    "2018-10-03",
+    "2018-10-09",
+    "2018-12-25",
+    "2018-12-31",
+    "2019-01-01",
+    "2019-02-04",
+    "2019-02-05",
+    "2019-02-06",
+    "2019-03-01",
+    "2019-05-01",
+    "2019-05-06",
+    "2019-06-06",
+    "2019-08-15",
+    "2019-09-12",
+    "2019-09-13",
+    "2019-10-03",
+    "2019-10-09",
+    "2019-12-25",
+    "2019-12-31"
+])
+
+
+class XKRXExchangeCalendar(PrecomputedTradingCalendar):
+    """
+    Calendar for the Korea exchange, and the primary calendar for
+    the country of South Korea.
+
+    Open Time: 9:00 AM, KST (Korean Standard Time)
+    Close Time: 3:30 PM, KST (Korean Standard Time)
+
+    Due to the complexity around the Korean holidays, we are hardcoding
+    a list of holidays covering 1986-2019, inclusive.
+    """
```

Comment:
I don't know if it's worth it, but maybe you could add the list of Korean holidays in this docstring, like is done in other calendars? Just the names of them.